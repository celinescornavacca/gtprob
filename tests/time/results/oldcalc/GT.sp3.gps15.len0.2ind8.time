0.46user 0.20system 0:00.87elapsed 77%CPU (0avgtext+0avgdata 30544maxresident)k
125inputs+0outputs (0major+12817minor)pagefaults 0swaps
