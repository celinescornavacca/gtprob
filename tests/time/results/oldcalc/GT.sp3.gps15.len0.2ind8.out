-113.704665268
